packages/app-desktop/gui/NoteEditor/utils/contextMenu.ts

```diff
@@ -106,8 +106,10 @@ export function menuItems(dispatch: Function): ContextMenuItems {
 				if (!options.filename) {
 					throw new Error('Filename is needed to save as png');
 				}
+				// double dimensions to make sure it's always big enough even on hdpi screens
+				const [width,height] = svgDimensions(document,options.textToCopy).map((x: number) => x * 2);
```

Comment:
> when it's undefined, what dimensions do we end up using?

The dimensions that used to be before this fix.
anything better we can choose as default ? 🤔

Reply:
How about the NaN dimensions, what do you plan to do about it?

> The dimensions that used to be before this fix.

Which are?

Reply:
> How about the NaN dimensions, what do you plan to do about it?

assigned dimensions as undefined if it's NaN

> Which are?

almost of zero use, but since we have no better fallback we have to use them if we can't find any sensible default value.
Dimensions were being calculated as follows,:
we are making an <img> element for the svg and using it's resolution to set exported image's resolution
img is a [replaced element](https://developer.mozilla.org/en-US/docs/Web/CSS/Replaced_element) and it's default size is 300 * 150 a/c to google search.
different sized mermaids i exported to check size were having height 150px each. and different heights all less than 300.

Reply:
Ok thanks for clarifying. The default dimensions indeed aren't useful so let's make them 1024. Is it possible at least to respect the diagram aspect ratio? Otherwise make it 1024x1024

Reply:
The max dimension would be 1024. For example 600x1024, or 1024x800